### PR TITLE
[14.0] [MIG] analytic_partner: Migration to v14.0

### PR DESCRIPTION
**Pull request by CarlosRoca13**
cc @Tecnativa TT31113

please @sergio-teruel @pedrobaeza review it

### CODE REVIEW

File: analytic_partner/readme/CONFIGURE.rst [hunk 1]
```diff
@@ -1,5 +1,4 @@
 To configure this module, you need to:
 
 #. Go to **Setting > Users > Users** and check you have as at least "Adviser"
-   in your user profile and have checked the "Show Full Accounting Features"
```
Comment (pedrobaeza) [inline analytic_partner/readme/CONFIGURE.rst]: AFAIK, you still need this one.

**Comment by CarlosRoca13** [comment]
@pedrobaeza changed

**Review by pedrobaeza — APPROVED** [review]

**Review by chienandalu — APPROVED** [review]
Rebase your branch to avoid runbot warnings (not this modules fault anyway)

**Comment by pedrobaeza** [comment]
The bot does the rebase:

/ocabot merge nobump

**Pull request merged**